### PR TITLE
Use a new regex for matching `/report` and `@admins`

### PR DESCRIPTION
**Pull request by vrumger**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -21,8 +21,8 @@ def main():
     dp = updater.dispatcher
     # group handlers
     dp.add_handler(MessageHandler(Filters.status_update.new_chat_members, group.added))
-    # thanks to https://t.me/dotvhs, who likes regex (weird). test: https://regex101.com/r/h4oH0X/2
-    dp.add_handler(MessageHandler(Filters.regex(r"(\s|^)(/report|@admins?)(\s|$)") & Filters.chat_type.groups,
+    # thanks to https://t.me/dotvhs and https://t.me/twitface, who like regex (weird). test: https://regex101.com/r/Amrm07/1
+    dp.add_handler(MessageHandler(Filters.regex(r"^/report\b|(^|\s)@admin[a-z_]{0,27}\b") & Filters.chat_type.groups,
```
Comment (Bibo-Joshi) [inline bot.py:25]: why `[a-z_]{0,27}` and not `[a-z_]*` or even `\w*`?

Comment (vrumger) [reply bot.py:25]: I didn't use `*` because Telegram has limit of 32 characters for usernames, and I don't remember why I didn't use `\w`.

Comment (Bibo-Joshi) [reply bot.py:25]: I see. but as this filter is not even using entities in the first place, it doesn't have to rely on `@admin…` being marked as username, right?

Comment (vrumger) [reply bot.py:25]: Yep.

**Pull request merged**